test/Tests/src/Data/Text/Encoding_Spec.enso

```diff
@@ -20,12 +19,12 @@ spec =
             invalid.to_java_charset . should_fail_with Illegal_Argument_Error
 
         Test.specify "Can get full set of character sets" <|
-            character_sets = Encoding_Module.all_character_sets
+            character_sets = all_character_sets
```

Comment:
This change seems inconsistent to me with the other one that adds the `Statistics.` prefix everywhere. I think we should go one way or the other (I prefer the more concise way, but that is just personal preference, justified by an opinion that IMO it is good for readability of the test code). But regardless of what we choose, I think we need to be consistent, unless there is some reason to do this differently here and there?